src/sentry/api/bases/organization.py

```diff
@@ -289,7 +289,9 @@ def convert_args(self, request, organization_slug, *args, **kwargs):
         except Organization.DoesNotExist:
             raise ResourceDoesNotExist
 
-        with sentry_sdk.start_span(op="check_object_permissions_on_organization"):
+        with sentry_sdk.start_span(
+            op="check_object_permissions_on_organization", description=organization_slug
```

Comment:
i would probably just use tags here since its always the same org given the event (and I think its already tagged)

Reply:
Yes, the organization is already tagged on the transaction. There is no actual need to capture the organization slug here.

My intent was to try to fill in a description wherever a sensible value was available, just to replace the hex ID in the Discover view with something more human-friendly. For someone reading the span row and trying to parse what it does, I figure listing out the function's main argument could provide a little cognitive boost (considering that the screen real estate would go to waste otherwise) even if it's not truly necessary.